ci-operator/step-registry/telcov10n/pre-ga-catalog/telcov10n-pre-ga-catalog-commands.sh

```
@@ -495,10 +251,62 @@ EOF
+  - mirrors:
+    - quay.io/prega/test/acm-d
+    source: registry.redhat.io/rhacm2
+EOF
+
+
+  echo "Creating Pre GA static catalog source..."
+
+  echo $catalogsources > ${prega_info_dir}/catalogSource.yaml
+  echo $imageDigestMirrorSet > ${prega_info_dir}/imageDigestMirrorSet.yaml
+  echo "Pre GA static catalog source has been created successfully!!!"
+}
```

Shellcheck (0.11.0)

[error] Unexpected keyword/token. Fix any mentioned problems and try again.

(SC1072)